### ISSUE TITLE
Validate strings with latin characters like Ñ

### ISSUE THREAD

**Issue by j2deme** [Respect/Validation#92]
I've used Validation mainly in English, but because of a project I'm doing I have to validate strings in spanish, meaning that the strings may include accents (áéíóúÁÉÍÓÚ), the letter Ñ (both lower and uppercase) and once in a while üÜ.

So my question is: Is there a way to validate this type of strings?

Besides using v::alpha or v::alnum and having to declare every additional chars.

``` php
v::alnum('áéíóúÁÉÍÓÚñÑ')->validate('La piñata de la nación'); 
```

**Comment by tsamu** [comment]
What about preg_match("/^[\pL]*$/", utf8_decode($val))

**Comment by henriquemoody** [comment]
I created #762 in order to fix this issue.

However, before I merge it I would like to understand in which scenario do you intent to use this validator?

**Comment by j2deme** [comment]
It's been a long time since I needed to use this feature and to be honest,
I don't remember the use case. So if nobody needs it, feel free to merge
and close this issue.

El mié., 16 de nov. de 2016 10:05 AM, Henrique Moody <
notifications@github.com> escribió:

I created #762 <https://github.com/Respect/Validation/pull/762> in order to
fix this issue.

However, before I merge it I would like to understand in which scenario do
you intent to use this validator?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Respect/Validation/issues/92#issuecomment-260986469>,
or mute the thread

**Comment by henriquemoody** [comment]
Now is more clear what is the intention here.

According to [Wikipedia](https://en.wikipedia.org/wiki/Alphanumeric):
> Alphanumeric is a combination of alphabetic and numeric characters, and is used to describe the collection of Latin letters and Arabic digits or a text constructed from this collection.

That said, the one we have is correct, and changing it would make it wrong. What is needed is a new rule, with a new name and a new purpose. I'm planning to finish it this week, and I'm gonna make it more clear on the documentation. The rule will be probably called _Name_ but I'm very open for suggestions.

**Comment by nickl-** [comment]
Closed: as per #762 Won't fix
